examples/state_and_process_plots.ipynb

```diff
@@ -0,0 +1,624 @@
+{
```

Comment:
This looks great! We should remember to strip the metadata/output before we merge.

I'm confused by the real_hinton plot--isn't the size and the color redundant? Don't we want the size to be the magnitude and the color to be the complex phase? Maybe I'll see later down.

Reply:
That is a great idea. I think we could modify the `hinton` code (rather than `hinton_real`) to achieve that goal.

You other point (from our offline discussion) about the incorrect behavior for values that are between the max and min values (evidenced by the small grey squares below) is a problem that I will fix now. 

![image](https://user-images.githubusercontent.com/2394563/57410910-75fc2900-722f-11e9-9ef2-3acc1efeb104.png)

Reply:
fixed